benchmarks/TerraFX.Interop.Mimalloc.AllocatorsBenchmarks.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <NoWarn>
+      $(NoWarn);
+
+      <!--Missing XML comments-->
+      CS1591
+    </NoWarn>
```

Comment:
All of this should be automatically coming from the central Directory.Build.props

Reply:
But that one does enforce all warnings, and it would be bad to make it for all projects (i. e. you do want to enforce it for the lib itself ; but not for the benchmarks). Or you mean to create a benchmarks-local build.props file, and put it there?

Reply:
Well everything except for `NoWarn` is already in https://github.com/terrafx/terrafx.interop.mimalloc/blob/main/Directory.Build.props

For `NoWarn` in particular, its fine for it to be here just like I do in the main source project: https://github.com/terrafx/terrafx.interop.mimalloc/blob/main/sources/Interop/Mimalloc/TerraFX.Interop.Mimalloc.csproj#L6

Reply:
But that's how I do it too, not sure if I get your point 😅

Reply:
This was specifically calling out that you're redeclaring `Nullable` and `AllowUnsafeBlocks`. You're also overriding the default `TreatWarningsAsErrors` behavior.

`NoWarn` itself is probably fine (minus doing it on multiple lines, since that's not really done elsewhere)